packs/android/config.yml

```diff
@@ -19,3 +19,4 @@ documentation:
       source software.
     url: >-
       https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android
+keywords: mobile, mobile agent
```

Comment:
YAML will treat this syntax as a string instead of a list. To make it a list/array, we will need to update it to this syntax
```yaml
keywords:
  - mobile
  - mobile agent
```

Reply:
if its easier, wrapping the values in brackets is also equivalent.

```yaml
keywords: [mobile, mobile agent]
```